### PR TITLE
Rename Attribute 4 to events log

### PR DESCRIPTION
**Pull request by hansl**
Its more general than a transaction log.

### CODE REVIEW

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl [hunk 1]
```diff
@@ -13,35 +13,35 @@ multisig-transaction-state /= 0  ; Pending
...
+event-info /= account-multisig-event-info-submit
+           /  account-multisig-event-info-approve
+           /  account-multisig-event-info-revoke
+           /  account-multisig-event-info-execute
+           /  account-multisig-event-info-withdraw
+           /  account-multisig-event-info-set-defaults
+           /  account-multisig-event-info-expire
+
+; Set of indices for multisig event entries.
+account-event-type-multisig-submit        = [9, 1, 0]
```
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:27]: Why did you change the format here?

Ref. spec:
```
; An attribute can have one or many attribute arguments.
attribute-arg = ()
attribute-id = uint
attribute = attribute-id / [attribute-id, * attribute-arg]
```

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl [hunk 2]
```diff
@@ -101,19 +101,19 @@ account-multisig-transaction-expire = {
...
-    /  { 0 => account-transaction-type-multisig-revoke, 1 => account.multisigRevoke@param }
-    /  { 0 => account-transaction-type-multisig-execute, 1 => account.multisigExecute@param }
-    /  { 0 => account-transaction-type-multisig-withdraw, 1 => account.multisigWithdraw@param }
-    /  { 0 => account-transaction-type-multisig-set-defaults, 1 => account.multisigSetDefaults@param }
+       { 0 => event-type-send, 1 => ledger.send@param }
+    /  { 0 => account-event-type-create, 1 => account.create@param }
+    /  { 0 => account-event-type-set-description, 1 => account.setDescription@param }
+    /  { 0 => account-event-type-add-roles, 1 => account.addRoles@param }
+    /  { 0 => account-event-type-remove-roles, 1 => account.removeRoles@param }
+    /  { 0 => account-event-type-delete, 1 => account.delete@param }
```
Comment (fmorency) [inline attributes/network/9_account_features/cddl/1_account_multisig.cddl:109]: You probably want to do it in another PR, but we need to update the spec to change `delete` with `disable` before merging https://github.com/liftedinit/many-framework/pull/140

**Review by fmorency — CHANGES_REQUESTED** [review]
Please address comments.

File: attributes/network/9_account_features/cddl/1_account_multisig.cddl (hunk 1 above)
Comment (hansl) [reply attributes/network/9_account_features/cddl/1_account_multisig.cddl:27]: Yeah I was going back and forth here, I guess I forgot to change it back.
